src/main/java/com/github/princesslana/eriscasper/util/SendableImage.java

```diff
@@ -0,0 +1,31 @@
+package com.github.princesslana.eriscasper.util;
+
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Base64;
+
+public final class SendableImage {
+
+  private SendableImage() {}
+
+  public static String imageToBase64(File file) throws IOException {
+    InputStream stream = file.toURI().toURL().openConnection().getInputStream();
```

Comment:
This stream is never closed?

Look into Guava's IO stuff: https://github.com/google/guava/wiki/IOExplained. I think you'll want to start with `Files.asByteSource`